internal/hud/webview/analytics_nudge.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func NeedsNudge(st store.EngineState) bool {
-	if st.AnalyticsOpt != analytics.OptDefault {
+	if st.AnalyticsUserOpt != analytics.OptDefault {
```

Comment:
probably doesn't matter much, but do you think it'd make sense to skip the nudge if the team is opted in?

Reply:
ooh, that's a good point. i could go either way on this, but ya you're right that skipping the nudge should make more sense